src/SE344/Views/Stocks/History.cshtml

```diff
@@ -0,0 +1,51 @@
+@{
+    ViewData["Title"] = "Stocks";
+    var transactions = ViewData["transactions"] as IEnumerable<KeyValuePair<string, StockTransaction>>;
+}
+<ul class="nav nav-tabs nav-justified">
+    <li>
+        <a asp-controller="Stocks" asp-action="Index">My Stocks</a>
+    </li>
+    <li class="active">
+        <a>History</a>
+    </li>
+    <li>
+        <a asp-controller="Stocks" asp-action="SearchStocks">Search</a>
+    </li>
+</ul>
+
+<div class="row">
+    <div class="col-md-10"> <!-- because why would putting the class on the table work? -->
+        <table id="historyTable" class="table">
+            <!-- `table.table` I feel dirty... -->
+            <thead>
+                <tr>
+                    <th>Stock</th>
+                    <th>Date</th>
+                    <th>Price per Share</th>
+                    <th>Num Shares</th>
+                    <th>Total Price</th>
+                </tr>
+            </thead>
+            <tbody>
+                @foreach (var t in transactions)
+                {
+                    <tr>
+                        <td>@t.Key</td>
+                        <td>@t.Value.TransactionDate</td>
+                        <td>@t.Value.PricePerShare</td>
+                        <td>@t.Value.NumShares</td>
+                        <td>@t.Value.TotalPrice</td>
+                    </tr>
+                }
+            </tbody>
+        </table>
+    </div>
+
+    <div id="historyActions" class="btn-group btn-group-vertical col-md-2">
+        <!-- TODO: horizontal alignment on tall, skinny screens -->
```

Comment:
I didn't say it was a :bug: though, which is why I mentioned "Enhancement".

I also didn't mean you had to do it right away. It was more of a reminder so feel free to wait until after this is merged to create the issue.